src/locale/en.json

```diff
@@ -129,6 +132,24 @@
         "lead": "We fully understand that the software has to solve business problems: optimize tasks and resources, secure data, enable further scaling, and make life easier. Nowadays, most or practically all processes taking place in companies are often based on IT systems. We want to help make sure everything works as well as possible."
       }
     },
+    "partnerships": {
+      "soda": {
+        "header": "Software Development Association Poland",
+        "lead": "Software Development Association Poland (SoDA) is an organization aimed at developing cooperation between software companies. It is the oldest and the largest organization of IT Service Employers in Poland, which is a mine of contacts and knowledge helping to develop the activities of companies operating in our industry. Thus, we are among the hundred most important companies on the market with a profile similar to ours."
+      },
+      "cwup": {
+        "header": "Collegium Witelona Uczelnia Państwowa",
+        "lead": "As part of the cooperation, we strive to cooperate in the improvement of study programs in the fields of Information Technology and Production and Logistics Engineering. In addition, we declare the possibility of carrying out practical classes (internships, projects), a constant willingness to accept students of selected fields of study for apprenticeships, and the possibility of cooperation in the development and implementation of educational projects."
```

Comment:
"As part of the cooperation, we strive to cooperate..."? The entire section looks like it was written in Polish and Google-translated into English. @jsawo @EwelinaLasowy maybe you could help here?

Reply:
Maybe something like: "As part of collaboration, we strive to cooperate..."?

Reply:
Changed on proposition of @EwelinaLasowy